src/components/ManageAccountsDrawer/components/LedgerOpenApp.tsx

```diff
@@ -110,18 +113,42 @@ export const LedgerOpenApp = ({ chainId, onClose, onNext }: LedgerOpenAppProps)
     if (isValidApp) {
       onNext()
     } else {
-      console.error('Correct Ledger app is not open', slip44Key)
+      toast({
+        title: translate('walletProvider.ledger.errors.appNotOpen', { app: slip44Key }),
+        description: translate('walletProvider.ledger.errors.appNotOpenDescription', {
+          app: slip44Key,
+        }),
+        status: 'error',
+        duration: 9000,
+        isClosable: true,
+        position: 'top-right',
+      })
     }
-  }, [isCorrectAppOpen, onNext, slip44Key])
+  }, [isCorrectAppOpen, onNext, slip44Key, toast, translate])
 
-  if (!asset) return null
+  const body = useMemo(() => {
+    if (!asset) return null
+    return (
+      <Center>
+        <Flex direction='column' justifyContent='center'>
+          <AssetOnLedger assetId={asset.assetId} size={'xl'} />
+          <RawText fontSize={'xl'} fontWeight={'bold'} mt={10} mb={3}>
+            {translate('accountManagement.ledgerOpenApp.title', {
+              chainNamespaceDisplayName: asset?.networkName ?? '',
```

Comment:
And as mentioned above, this won't be correct for EVM chains other than Ethereum